substreams/src/pb/foundational_store.rs

```diff
@@ -0,0 +1,99 @@
+pub struct GetRequest {
+    #[prost(uint64, tag = "1")]
+    pub block_number: u64,
+    #[prost(bool, tag = "3")]
+    pub omit_deleted: bool,
+    #[prost(bytes = "vec", tag = "4")]
+    pub key: ::prost::alloc::vec::Vec<u8>,
+}
+#[allow(clippy::derive_partial_eq_without_eq)]
+#[derive(Clone, PartialEq, ::prost::Message)]
+pub struct GetResponse {
+    #[prost(enumeration = "ResponseCode", tag = "4")]
+    pub response: i32,
+    #[prost(message, optional, tag = "5")]
+    pub value: ::core::option::Option<::prost_types::Any>,
+}
+#[allow(clippy::derive_partial_eq_without_eq)]
+#[derive(Clone, PartialEq, ::prost::Message)]
+pub struct GetAllRequest {
+    #[prost(uint64, tag = "1")]
+    pub block_number: u64,
+    #[prost(bool, tag = "3")]
+    pub omit_deleted: bool,
+    #[prost(bytes = "vec", repeated, tag = "4")]
+    pub keys: ::prost::alloc::vec::Vec<::prost::alloc::vec::Vec<u8>>,
+}
+#[allow(clippy::derive_partial_eq_without_eq)]
+#[derive(Clone, PartialEq, ::prost::Message)]
+pub struct ResponseEntry {
+    #[prost(bytes = "vec", tag = "1")]
+    pub key: ::prost::alloc::vec::Vec<u8>,
+    #[prost(message, optional, tag = "2")]
+    pub response: ::core::option::Option<GetResponse>,
+}
+#[allow(clippy::derive_partial_eq_without_eq)]
+#[derive(Clone, PartialEq, ::prost::Message)]
+pub struct GetAllResponse {
+    #[prost(message, repeated, tag = "1")]
+    pub entries: ::prost::alloc::vec::Vec<ResponseEntry>,
+}
+#[derive(Clone, Copy, Debug, PartialEq, Eq, Hash, PartialOrd, Ord, ::prost::Enumeration)]
+#[repr(i32)]
+pub enum ResponseCode {
+    Unset = 0,
+    Found = 1,
+    NotFound = 2,
+    NotFoundFinalize = 4,
+}
+impl ResponseCode {
+    /// String value of the enum field names used in the ProtoBuf definition.
+    ///
+    /// The values are not transformed in any way and thus are considered stable
+    /// (if the ProtoBuf definition does not change) and safe for programmatic use.
+    pub fn as_str_name(&self) -> &'static str {
+        match self {
+            ResponseCode::Unset => "UNSET",
+            ResponseCode::Found => "FOUND",
+            ResponseCode::NotFound => "NOT_FOUND",
+            ResponseCode::NotFoundFinalize => "NOT_FOUND_FINALIZE",
+        }
+    }
+    /// Creates an enum from field names used in the ProtoBuf definition.
+    pub fn from_str_name(value: &str) -> ::core::option::Option<Self> {
+        match value {
+            "UNSET" => Some(Self::Unset),
+            "FOUND" => Some(Self::Found),
+            "NOT_FOUND" => Some(Self::NotFound),
+            "NOT_FOUND_FINALIZE" => Some(Self::NotFoundFinalize),
+            _ => None,
+        }
+    }
+}
+#[allow(clippy::derive_partial_eq_without_eq)]
+#[derive(Clone, PartialEq, ::prost::Message)]
+pub struct AccountOwner {
+    #[prost(bytes = "vec", tag = "1")]
+    pub mint: ::prost::alloc::vec::Vec<u8>,
+    #[prost(bytes = "vec", tag = "2")]
+    pub owner: ::prost::alloc::vec::Vec<u8>,
+}
+// @@protoc_insertion_point(module)
```

Comment:
This doesn't belong to a "base" foundational store protobuf.

Moreover, by the filename, I have the impression that Protobuf package is not namespaced correctly. Indeed, if you check other existing, you will see their name looks like `./sf.substreams.v1.rs`, I would have expected to be `sf.substreams.foundational_store.v1` (wondering even if we should use `sf.substreams.foundational.store.v1`.